keps/976-plain-pods/README.md

```diff
@@ -372,12 +371,11 @@ matchExpressions:
 Once the webhook has marked Pods subject to queuing with the `kueue.x-k8s.io/managed: true` label,
 the Pod reconciler can create the corresponding Workload object to feed the Kueue admission logic.
 
-Note that the Workload cannot be owned by the Pod. Otherwise any cascade deletion of the Pod would
-delete the Workload object, even before the Pod terminates (if it has a grace period).
-This means that the controller needs to manually delete the Workload object once it has the
-Finished condition (after we have determined that all pods have finished).
+The Workload will be owned by all Pods. Once all the Pods that own the workload are deleted (and
+their finalizers are removed), the Workload will be automatically cleaned up.
 
-A possible extension here is to add a TTL, which we can consider based on user feedback.
+If individual Pods in the group fail and a replacement Pod comes in, the replacement Pod will be
+added as an owner of the Workload as well.
```

Comment:
What should be done to a failed pod when it's replaced? Should it be deleted/finalized?

Do we only support replacement for failed pods?

Reply:
Nothing is done to a Failed pod until the Workload is declared finished.

Yes, we only support replacement for Failed Pods. Although this is implicit, as we just don't count failed pods towards the podsets. We do for Succeeded Pods.